src/components/PostLoginLayout.tsx

```diff
@@ -1,8 +1,18 @@
 import { Outlet } from 'react-router-dom';
+
 import styled from 'styled-components';
+
+import useCheckAccessToken from '../hooks/useCheckAccessToken';
+
 import Header from './Header';
 
 export default function PostLoginLayout() {
+  const ready = useCheckAccessToken();
+
+  if (!ready) {
+    return null;
+  }
+
```

Comment:
p2: 동료 개발자가 `useCheckAccessToken`이라는 훅을 사용할 때 `기존에 이 훅을 만드신 분은 변수 네이밍을 뭘로 했지?` 하면서 컨벤션을 맞추려는 비용이 들 것 같은데요. hook에서 객체를 리턴해줘서 아래와 같이 사용하면 훨씬 편리할 것 같은데 어떠신가요? 추가적으로 `useCheckAccessToken`에 변화가 생겨 return해야 하는 값이 추가되어도 수정할 필요가 없을 것 같아요.

```suggestion
  const { ready } = useCheckAccessToken();

  if (!ready) {
    return null;
  }

```